tools/a-star/src/pictureMapper/GetPixelColor.java

```diff
@@ -0,0 +1,94 @@
+package pictureMapper;
+
+import javax.imageio.ImageIO;
+import java.awt.image.BufferedImage;
+import java.awt.image.DataBufferByte;
+import java.io.File;
+import java.io.IOException;
+
+/**
+ * This class and driver is used to transform Maps(images) to Boolean Arrays
+ * You can use this driver to scale images using scalePNG("path-of-image", "where-to-save-with filename", "scale to size")
+ */
+
+//-2476490 == dark red (classroom)
+//-534826 == light red (corridor)
+
+public class GetPixelColor {
+    public static void main(String[] args) throws IOException {
+
+
+        scalePNG("media/h9.png", "media/h9275.png", 275);
+
```

Comment:
I dont think this main is necessary anymore

Reply:
Yes, it is as this will be used for the other pictures. They also serve as a template if someone else wants to use it.